numpy/ma/tests/test_core.py

```diff
@@ -4826,6 +4826,16 @@ def test_copy(self):
             np.ma.masked.copy() is np.ma.masked,
             np.True_.copy() is np.True_)
 
+    def test__copy(self):
+        assert_equal(
+            copy.copy(np.ma.masked) is np.ma.masked,
+            copy.copy(np.True_) is np.True_)
+
+    def test__deepcopy(self):
```

Comment:
I'd replace this with just `test_deepcopy`, dropping the extra underscore

Reply:
I'll make the changes and commit

Reply:
Using the online editor to make these changes is alright?

Reply:
Yep, online editor is fine - I can just squash the commits together when I merge.

For bigger patches that make sense as multiple commits, it's better to squash by hand offline, but there's no need to here.